readthedocs/projects/management/commands/fixtures_projects.py

```diff
@@ -0,0 +1,48 @@
+"""
+Fixture Projects.
+
+This command intends to replace the fixtures file
+which is used and instead create projects directly.
+
+"""
+
+from django.core.management.base import BaseCommand
+
+from readthedocs.projects.models import Project
+
+
+class Command(BaseCommand):
+
+    help = __doc__
+
+    def load_data(self):
+        projects = [
+            ("Read The Docs", "https://github.com/rtfd/readthedocs.org"),
+            ("Django Test Utils", "https://github.com/ericholscher/django-test-utils"),
+            ("Fabric", "https://github.com/fabric/fabric"),
+            ("South", "https://github.com/dmishe/django-south"),
+            ("Pip", "https://github.com/pypa/pip"),
+            ("Kong", "https://github.com/ericholscher/django-kong"),
+            ("Djangoembed", "https://github.com/worldcompany/djangoembed"),
+            ("Tastypie", "https://github.com/codysoyland/django-tastypie"),
+            ("Django Uni Form", "https://github.com/pydanny/django-uni-form"),
+            ("Cue", "https://github.com/coleifer/cue.git"),
+            ("Pinax", "https://github.com/pinax/pinax"),
+            ("Haystack", "https://github.com/toastdriven/django-haystack"),
+            ("Taggit", "https://github.com/alex/django-taggit"),
+            ("test_project", "https://github.com/bogususer/non-existing-repo"),
+            ("Sphinx", "https://github.com/sphinx-doc/sphinx"),
+            ("Numpy", "https://github.com/numpy/numpy"),
+            ("Conda", "https://github.com/conda/conda"),
+        ]
```

Comment:
For the tests that rely on specific Build objects to exist, we can add a minimal dictionary for each project containing data for extra Build objects to create for the project.

Unfinished example:

```
       projects = [
            # ...
            ("Pip", "https://github.com/pypa/pip", {"slug": "0.8", "identifier": "2404a34eba4ee9c48cc8bc4055b99a48354f4950"}),
            # ...
        ]

```